cmd/object-api-datatypes.go

```diff
@@ -59,6 +59,12 @@ type BucketInfo struct {
 	Created time.Time
 }
 
+// HealInfo - represents erasure coded chunks in disks
+// TBD: Final choice of datatype
+type HealInfo struct {
+	Shards []bool
```

Comment:
We also need to add meaningful info like "Depending" on the shards can this object healed or not on what state the object is.  This would allow `mc` to make intelligent decisions. This value is needed but might not be readily consumable the client and make decisions.